examples/data-objects/todo/src/index.tsx

```diff
@@ -3,12 +3,46 @@
  * Licensed under the MIT License.
  */
 
-import { ContainerRuntimeFactoryWithDefaultDataStore } from "@fluidframework/aqueduct";
+import { IContainerContext } from "@fluidframework/container-definitions";
+import { ContainerRuntime } from "@fluidframework/container-runtime";
+import { buildRuntimeRequestHandler } from "@fluidframework/request-handler";
+import { IContainerRuntimeBase } from "@fluidframework/runtime-definitions";
+import { RuntimeFactoryHelper } from "@fluidframework/runtime-utils";
+import { defaultRouteRequestHandler } from "@fluidframework/aqueduct";
+import { IRequest } from "@fluidframework/core-interfaces";
 import { TodoInstantiationFactory, TodoName } from "./Todo";
 
-export const fluidExport = new ContainerRuntimeFactoryWithDefaultDataStore(
```

Comment:
I don't think you need to move away from `ContainerRuntimeFactoryWithDefaultDataStore` -- if you pass the `innerRequestHandler` to the 4th argument of the constructor I think it should work fine?

Reply:
Passed innerRequestHandler as 4th parameter. Initially, it was messing up the example because of the reordering on request handlers. 

Similarly for spaces